### PR TITLE
refactor: use jina commons

### PR DESCRIPTION
**Pull request by slettner**
Issue: https://github.com/jina-ai/jina-commons/issues/6

### CODE REVIEW

File: vggish_audio_encoder.py [hunk 1]
```diff
@@ -44,33 +45,35 @@ def __init__(self,
...
-
-        return filtered_docs
-
     def _create_embeddings(self, filtered_docs: Iterable):
         """Update the documents with the embeddings generated by VGGISH"""
 
         for d in filtered_docs:
             # Vggish broadcasts across different length audios, not batches
-            [embedding] = self.sess.run([self.embedding_tensor], feed_dict={self.feature_tensor: d.blob})
+            print('###', len(d))
```
Comment (JoanFM) [inline vggish_audio_encoder.py]: remove this

File: vggish_audio_encoder.py [hunk 2]
```diff
@@ -44,33 +45,35 @@ def __init__(self,
...
+            [embedding] = self.sess.run([self.embedding_tensor], feed_dict={self.feature_tensor: d[0].blob})
             result = self.post_processor.postprocess(embedding)
-            d.embedding = (np.float32(result) - 128.) / 128.
+            d[0].embedding = (np.float32(result) - 128.) / 128.
```
Comment (JoanFM) [inline vggish_audio_encoder.py]: this makes no sense, what is d[0]?

File: vggish_audio_encoder.py [hunk 3]
```diff
@@ -44,33 +45,35 @@ def __init__(self,
...
-        return filtered_docs
-
     def _create_embeddings(self, filtered_docs: Iterable):
         """Update the documents with the embeddings generated by VGGISH"""
 
         for d in filtered_docs:
             # Vggish broadcasts across different length audios, not batches
-            [embedding] = self.sess.run([self.embedding_tensor], feed_dict={self.feature_tensor: d.blob})
+            print('###', len(d))
+            [embedding] = self.sess.run([self.embedding_tensor], feed_dict={self.feature_tensor: d[0].blob})
```
Comment (JoanFM) [inline vggish_audio_encoder.py]: same here d[0]????

**Review by JoanFM — DISMISSED** [review]

File: vggish_audio_encoder.py (hunk 3 above)
Comment (slettner) [reply vggish_audio_encoder.py]: yes, it looks weird. The reason is that the generator yields batches, i.e. `DocumentArray`.
But vggish does not broadcast batches so we need to use batch_size=1 and take d[0].

File: vggish_audio_encoder.py (hunk 1 above)
Comment (slettner) [reply vggish_audio_encoder.py]: 59e40f3313db43e469e957aff1e7f294e2c8916f

File: vggish_audio_encoder.py (hunk 3 above)
Comment (slettner) [reply vggish_audio_encoder.py]: Maybe for executors that don't support batching the function is not that useful.
Or I can rename `d` to `doc_batch` and add a comment for clarification?

What do you think?

Comment (JoanFM) [reply vggish_audio_encoder.py]: if there is no batch being used there is no point to use the function

Comment (slettner) [reply vggish_audio_encoder.py]: agree

Comment (slettner) [reply vggish_audio_encoder.py]: https://github.com/jina-ai/executor-audio-VGGishEncoder/pull/7/commits/de64fa42a400f6034ed6866931542e30ac905ecd

Comment (slettner) [reply vggish_audio_encoder.py]: Removed the function. Not closing PR because  docstrings were missing and are worth adding

**Review by nan-wang — DISMISSED** [review]
LGTM👍

**Review by Yongxuanzhang — APPROVED** [review]

**Pull request merged**